server/db.go

```diff
@@ -77,16 +80,14 @@ func DbConnect(ctx context.Context, logger *zap.Logger, config Config, create bo
 		logger.Fatal("Failed to open database", zap.Error(err))
 	}
 
+	dbPing(ctx, logger, db, dbName)
+
 	if create {
 		var nakamaDBExists bool
 		if err = db.QueryRow("SELECT EXISTS (SELECT 1 from pg_database WHERE datname = $1)", dbName).Scan(&nakamaDBExists); err != nil {
-			var pgErr *pgconn.PgError
-			if errors.As(err, &pgErr) && pgErr.Code == dbErrorDatabaseDoesNotExist {
-				nakamaDBExists = false
-			} else {
-				db.Close()
-				logger.Fatal("Failed to check if db exists", zap.String("db", dbName), zap.Error(err))
-			}
+			nakamaDBExists = false
```

Comment:
This line can be removed as the logger.Fatal will terminate the execution anyways